scripts/swolisten.c

```diff
@@ -435,10 +435,36 @@ int usbFeeder(void)
       if (!(dev = libusb_get_device(handle)))
 	continue;
 
-      if (libusb_claim_interface (handle, INTERFACE)<0)
+      uint8_t bmp_interface = 0;
+      unsigned char bmp_endpoint = 0;
+
+      if (libusb_get_active_config_descriptor(dev, &config) < 0)
+	continue;
+
+      int interface_found = 0;
+      for (int if_num = 0; if_num < config->bNumInterfaces && !interface_found; if_num++) {
```

Comment:
Given `bNumInterfaces` is a uint8_t in the config descriptors structure, please use `uint8_t` for `if_num` here.

Reply:
Accepted. It's not like STM32F1 fsdev or STM32F4 dwc2 OTG_FS can support more than 8 endpoints, so I can't imagine >256 interfaces being plausible.